andela/mrm-mobile
# #159276735 Standardize position and type of close buttons.

#### What does this PR do?

Standardize position and type of close buttons and style icon capacity in 7 inch tablet.

#### How should this be manually tested?

* Run the application.
* Navigate to the Lagos meeting room.
* Check the Find Room, Schedule and Room Info pages, to view the update.

#### Any background context you want to provide?

* The buttons were inconsistent and positioned differently.
* Capacity icon didn't match as in the 10 Inch tablet.

#### What are the relevant pivotal tracker stories?

[#159276735](https://www.pivotaltracker.com/story/show/159276735)

#### Screenshots (if appropriate)

<img width="685" alt="screen shot 2018-07-25 at 02 05 51" src="https://user-images.githubusercontent.com/15923109/43171037-d738a3fe-8fb0-11e8-9126-f13dcccff2c5.png">

* Capacity Icon style

<img width="613" alt="screen shot 2018-07-25 at 02 05 23" src="https://user-images.githubusercontent.com/15923109/43171044-e41371c6-8fb0-11e8-8a7e-2602437a102a.png">
